### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by ritazakhodyaeva**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
 <!-- default badges end -->
 # Dashboard for WinForms - Inspect the Dashboard Performance
 
-
+The following example shows how to integrate the Dashboard Diagnostic Tool's functionality in the Dashboard Designer to allow users to inspect the dashboard's performance.
 
 ## Overview
 
-
+The **Inspect** button is inserted to the custom Ribbon's **Performance Diagnostics** page group. The button click starts a new diagnostic session. The Dashboard Diagnostic Tool monitors user actions to collects actions for each event of the event session. When you uncheck the button, the session ends and the resulting report is automatically saved to the XML file. You can open the report in the Diagnostic Tool UI.
```
Comment (albertov05) [inline readme.md]:
```suggestion
The **Inspect** button is inserted into the custom Ribbon's **Performance Diagnostics** page group. The button click starts a new diagnostic session. The Dashboard Diagnostic Tool monitors user actions to collects actions for each event of the event session. When you uncheck the button, the session ends and the resulting report is automatically saved to the XML file. You can open the report in the Diagnostic Tool UI.
```

File: readme.md [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
 
 ![Dashboard Diagnostic Tool integrated to the Dashboard Designer](./images/dashboardMain.png)
```
Comment (albertov05) [inline readme.md]:
```suggestion
![Dashboard Diagnostic Tool integrated into the Dashboard Designer](./images/dashboardMain.png)

```

File: readme.md [hunk 3]
```diff
@@ -19,7 +18,19 @@
 
 ## Inspect the Dashboard Performance in the Dashboard Designer
 
+1. Download the [Dashboard Diagnostic Tool](https://github.com/DevExpress-Examples/bi-dashboard-diagnosic-tool). 
+
+2. Reference `DiagnosticTool.dll` and install the [Microsoft.Diagnostics.Tracing.TraceEvent](https://www.nuget.org/packages/Microsoft.Diagnostics.Tracing.TraceEvent/) package in your dashboard project. 
+
+3. Create a custom button and insert it to the Ribbon.
```
Comment (albertov05) [inline readme.md]:
```suggestion
3. Create a custom button and insert it in the Ribbon.

```

File: readme.md [hunk 4]
```diff
@@ -19,7 +18,19 @@
...
 
+1. Download the [Dashboard Diagnostic Tool](https://github.com/DevExpress-Examples/bi-dashboard-diagnosic-tool). 
+
+2. Reference `DiagnosticTool.dll` and install the [Microsoft.Diagnostics.Tracing.TraceEvent](https://www.nuget.org/packages/Microsoft.Diagnostics.Tracing.TraceEvent/) package in your dashboard project. 
+
+3. Create a custom button and insert it to the Ribbon.
+
+4. Create a `DiagnosticController` object. 
+
+5. Call the conroller's Start() and Stop() methods on button's click to run and finish the Dashboard Diagnostic Tool's session.
```
Comment (albertov05) [inline readme.md]: Note that "conroller's" is mispelled.
Also, what do you mean by "on button's click"? For example, do you mean:

`5. Call the controller's Start() and Stop() methods on a button click to run and finish the Dashboard Diagnostic Tool's session.`

or

`5. Call the controller's Start() and Stop() methods on each button click to run and finish the Dashboard Diagnostic Tool's session.`

File: readme.md [hunk 5]
```diff
@@ -19,7 +18,19 @@
...
+
+6. Implement the `IFileController` interface and specify the output file path in the `TrySaveFile` method. Pass a new class instance that implements `IFileController` to the controller's contructor.
```
Comment (albertov05) [inline readme.md]: 1. "contructor" is misspelled. Please change it to "constructor".

2. I don't think "implements `IFileController` _to_" is correct.
Do you mean "implements `IFileController` _in_ the controller's constructor"?

File: readme.md (hunk 4 above)
Comment (ritazakhodyaeva) [reply readme.md]:
```suggestion
5. Call the controller's Start() and Stop() methods on a button click to run and finish the Dashboard Diagnostic Tool's session.

```

File: readme.md (hunk 5 above)
Comment (ritazakhodyaeva) [reply readme.md]:
```suggestion
6. Implement the `IFileController` interface and specify the output file path in the `TrySaveFile` method. Pass a new class instance that implements `IFileController` in the controller's constructor. 

```

**Pull request merged**